iphone/Maps/Core/Theme/Extensions/UIColor+hexString.swift

```diff
@@ -0,0 +1,23 @@
+extension UIColor {
+    var hexString: String {
+      let cgColorInRGB = cgColor.converted(to: CGColorSpace(name: CGColorSpace.sRGB)!, intent: .defaultIntent, options: nil)!
+              let colorRef = cgColorInRGB.components
```

Comment:
Code formatting looks wrong and does not conform to OM style.

Reply:
fixed

Reply:
how about adding of https://github.com/realm/SwiftLint ?